pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/kits/KitRequestShippingTest.java

```diff
@@ -66,6 +72,13 @@ public static void doFirst() {
         notLegacyParticipant = TestParticipantUtil.createParticipantWithEsProfile(notLegacyParticipantGuid, profile, ddpInstanceDto);
         notLegacyParticipantGuid = notLegacyParticipant.getRequiredDdpParticipantId();
         participants.add(notLegacyParticipant);
+
+        Profile mimicNotHuridParticipant = new Profile();
```

Comment:
just a heads up - hruid is written as hurid here

Reply:
Oops thanks!!!